README.md

```diff
@@ -61,6 +61,59 @@ We need to change the container IP address because it should take part of the sa
 To allow the access from your container to you NVIDIA card, you must to create a new lxd profile and assign it to your robocup container.
 [Follow this guide to create the profile](https://blog.simos.info/how-to-easily-run-graphics-accelerated-gui-apps-in-lxd-containers-on-your-ubuntu-desktop/)
```

Comment:
With these changes, are we still need to follow the tutorial? If not, remove it please.

Reply:
That link and the profile given there worked well in my computer with a AMD Radeon graphics card. I'll make it clear